docs/modules/plugins/partials/plugin-json-matchers.adoc

```diff
@@ -0,0 +1,58 @@
+== Matchers
+Matcher is an object which compares objects against a specific criteria and returns a boolean value indicating whether the match was successful or not.
+
+Embedded matchers, placeholders and expressions can be found on the json-unit library https://github.com/lukas-krecan/JsonUnit#readme[readme].
+
+=== Array contains any of entries
+
+Validate is actual array contains any of entries from the expected one.
```

Comment:
is -> if ?

Reply:
It's definitely not correct, there is no such pairing as "is <...> contains"

Reply:
google. agree